src/store/modules/annotation/actions.js

```diff
@@ -6,9 +6,11 @@ export default {
     dispatch,
     commit
   }, payload) => {
+    paper.view.autoUpdate = 
```

Comment:
Is there something missing here? 
Should it be: `paper.view.autoUpdate = false` ?